src/bin/worker.js

```diff
@@ -1,27 +1,25 @@
-var Docker = require('../lib/docker/docker');
-var fs = require('fs');
-var os = require('os');
-var path = require('path');
-var program = require('commander');
-var taskcluster = require('taskcluster-client');
-var base = require('taskcluster-base');
-var createLogger = require('../lib/log').createLogger;
-var debug = require('debug')('docker-worker:bin:worker');
-var _ = require('lodash');
-var monitoring = require('taskcluster-lib-monitor');
-
-var Runtime = require('../lib/runtime');
-var TaskListener = require('../lib/task_listener');
-var ShutdownManager = require('../lib/shutdown_manager');
-var GarbageCollector = require('../lib/gc');
-var VolumeCache = require('../lib/volume_cache');
-var PrivateKey = require('../lib/private_key');
-var reportHostMetrics = require('../lib/stats/host_metrics');
-var ImageManager = require('../lib/docker/image_manager');
-
+import 'source-map-support/register'
```

Comment:
I think you typically add this to built files rather that source files, but I may be wrong.